www/README.md

```diff
@@ -27,27 +27,43 @@ We aim to follow the [Divio documentation system](https://documentation.divio.co
 For reference documentation (APIs, configuration, etc.), use the following format:
 
 ```
-method/name | `type` | Default: `value` : description
+method/name | `type` |  Default: `value` | _required_ : description
 ```
 
-`Default` is optional and only applies if there is a default.
+`Default` is optional and only applies if there is a default. `_required_` can be omitted if the value is optional.
 
 Example:
 
 ```
 devOptions.open | `string` | Default: `"default"`
 ```
 
-If it's a heading use the truncated format with the rest of the information below the heading:
+If it's a heading use the extended format with the rest of the information below the heading:
 
 ```
 #### name
-`string`
-Default: `value`
-Description
+**Type:** `string`
+
+**Default:** `value`
+
+_required_
+
+Short 1-2 sentence description ideally the same as what's in the code comments.
+
+Details
+
+Example(s):
 ```
 
-Examples:
+#### Types
```

Comment:
@FredKSchott your input on this is appreciated, not sure what direction to go in